src/Mailbox.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Controls from './MailboxControls';
+import Messages from './MailboxMessages';
+import Attachments from './MailboxAttachments';
+import Attachment from './MailboxAttachment';
+import ReadMessage from './MailboxReadMessage';
+import ReadInfo from './MailboxReadInfo';
+
+export default {
+  Controls,
+  Messages,
+  Attachments,
+  Attachment,
```

Comment:
Would it make more sense to have this as a child of `Attachments`, `Item` maybe? So it would read `Mailbox.Attachments.Item`.